dissect/cstruct/cstruct.py

```diff
@@ -184,6 +209,8 @@ def addtype(self, name: str, type_: BaseType, replace: bool = False) -> None:
 
         self.typedefs[name] = type_
 
+    addtype = add_type
```

Comment:
`addtype` is only used in the basic tests and etl, so why not just update the name there?

test_basic.py line 42,44
etl: manifest.py line 106,107